src/Analyzer.JsonRuleEngine/JsonRuleEngine.cs

```diff
@@ -21,62 +21,98 @@ public class JsonRuleEngine : IRuleEngine
         /// </summary>
```

Comment:
`using Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.Expressions;` can be removed

Reply:
Ah, missed one